expected/circle.out

```diff
@@ -5,6 +5,22 @@
 (1 row)
 
 -- Input/Output ---
+SELECT scircle_deg(spoint(10,10), 90);
+               scircle_deg                
+------------------------------------------
+ <(0.57522204 , -0.57522204) , 1.5707963>
+(1 row)
+
+SELECT scircle_deg(spoint(10,10), 91);
+ERROR:  radius must be not greater than 90 degrees or less than 0
```

Comment:
This isn't part to your PR technically, but this error message is not grammatically correct. It should be "radius must not **be** greater than 90 degrees or less than 0".